### ISSUE TITLE
Make EightBallController#getMessages return xml

### ISSUE THREAD

**Issue by SilverPhoenix99** [qqupp/scalastuff#2]
It's currently returning text, and it would be a good move forward to building the api if it returns xml.

**Comment by qqupp** [comment]
Shall we go for content negotiation instead of passing a parameter in the url as we were discussing?
 https://playframework.com/documentation/2.6.x/ScalaContentNegotiation

**Comment by SilverPhoenix99** [comment]
I agree with you. How complex would it be to have both, with priority on the query parameter?
